scripts/jenkins-report.sh

```diff
@@ -14,6 +15,10 @@ source scripts/jenkins-common.sh
 # Get the diff coverage and html reports for unit tests
 paver coverage -b $TARGET_BRANCH
 
+if [[ -n "$PYTEST_CONTEXTS" ]]; then
```

Comment:
^ might be a little after the fact, but why is it `PTEST_CONTEXTS` and not `WHO_TESTS_WHAT`? The variable name is not-intuitive

Reply:
There are actually two things going on. First, we have to collect the `WHO_TESTS_WHAT` data (the pytest contexts that cover each line), and second, we have to use that to to only execute a subset of tests. I decided to drive each of those features w/ a separate environment variable for clarity.

That said, I'd be fine w/ renaming the env variables to something like `COLLECT_WHO_TESTS_WHAT` and `SELECT_WHO_TESTS_WHAT` or something like that.

Reply:
Sure, having two makes sense, but I would prefer it if the name was more related to the whole WTW task.